cclient/iota_api.c

```diff
@@ -30,6 +30,23 @@ iota_api_result_t iota_api_get_node_info(
 iota_api_result_t iota_api_get_neighbors(
     const iota_http_service_t* const service, get_neighbors_res_t* const res) {
   iota_api_result_t result = {0};
+
+  char_buffer_t* req_buff = char_buffer_new();
+  char_buffer_t* res_buff = char_buffer_new();
+
+  result.error = service->serializer.vtable.get_neighbors_serialize_request(
+      &service->serializer, req_buff);
+  if (result.error != RC_OK) goto done;
```

Comment:
add brackets

Reply:
what is the purpose of adding brackets in a single statement in the if condition?

Reply:
It's more readable, less error-prone, and it match the codebase style